EveripediaNetwork/ep-ui
# fixed bug with wiki report submission

# Wiki report submission bug

_Fixed bug with wiki report submission_

## How should this be tested?

1. Click on "Report to us" button from wiki details
2. Write in some texts then click "Confirm"

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/2911
